diff --git a/Kvantum/Matcha-Aliz/Matcha-Aliz.kvconfig b/Kvantum/Matcha-Aliz/Matcha-Aliz.kvconfig
index ca29b7b..839ddbb 100644
--- a/Kvantum/Matcha-Aliz/Matcha-Aliz.kvconfig
+++ b/Kvantum/Matcha-Aliz/Matcha-Aliz.kvconfig
@@ -61,7 +61,7 @@ dialog_button_layout=0
 intensity=1.00
 no_inactiveness=false
 no_window_pattern=false
-opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx
+opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx,calamares
 reduce_menu_opacity=0
 reduce_window_opacity=0
 respect_DE=true
diff --git a/Kvantum/Matcha-Azul/Matcha-Azul.kvconfig b/Kvantum/Matcha-Azul/Matcha-Azul.kvconfig
index 6786273..203af5b 100644
--- a/Kvantum/Matcha-Azul/Matcha-Azul.kvconfig
+++ b/Kvantum/Matcha-Azul/Matcha-Azul.kvconfig
@@ -61,7 +61,7 @@ dialog_button_layout=0
 intensity=1.00
 no_inactiveness=false
 no_window_pattern=false
-opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx
+opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx,calamares
 reduce_menu_opacity=0
 reduce_window_opacity=0
 respect_DE=true
diff --git a/Kvantum/Matcha-Dark-Aliz/Matcha-Dark-Aliz.kvconfig b/Kvantum/Matcha-Dark-Aliz/Matcha-Dark-Aliz.kvconfig
index b3f6f67..5c07338 100644
--- a/Kvantum/Matcha-Dark-Aliz/Matcha-Dark-Aliz.kvconfig
+++ b/Kvantum/Matcha-Dark-Aliz/Matcha-Dark-Aliz.kvconfig
@@ -61,7 +61,7 @@ dialog_button_layout=0
 intensity=1.00
 no_inactiveness=false
 no_window_pattern=false
-opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx
+opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx,calamares
 reduce_menu_opacity=0
 reduce_window_opacity=0
 respect_DE=true
diff --git a/Kvantum/Matcha-Dark-Azul/Matcha-Dark-Azul.kvconfig b/Kvantum/Matcha-Dark-Azul/Matcha-Dark-Azul.kvconfig
index 69a1384..da9c857 100644
--- a/Kvantum/Matcha-Dark-Azul/Matcha-Dark-Azul.kvconfig
+++ b/Kvantum/Matcha-Dark-Azul/Matcha-Dark-Azul.kvconfig
@@ -61,7 +61,7 @@ dialog_button_layout=0
 intensity=1.00
 no_inactiveness=false
 no_window_pattern=false
-opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx
+opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx,calamares
 reduce_menu_opacity=0
 reduce_window_opacity=0
 respect_DE=true
diff --git a/Kvantum/Matcha-Dark/Matcha-Dark.kvconfig b/Kvantum/Matcha-Dark/Matcha-Dark.kvconfig
index f3f6c97..9ec707d 100644
--- a/Kvantum/Matcha-Dark/Matcha-Dark.kvconfig
+++ b/Kvantum/Matcha-Dark/Matcha-Dark.kvconfig
@@ -61,7 +61,7 @@ dialog_button_layout=0
 intensity=1.00
 no_inactiveness=false
 no_window_pattern=false
-opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx
+opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx,calamares
 reduce_menu_opacity=0
 reduce_window_opacity=0
 respect_DE=true
diff --git a/Kvantum/Matcha-Light-Aliz/Matcha-Light-Aliz.kvconfig b/Kvantum/Matcha-Light-Aliz/Matcha-Light-Aliz.kvconfig
index 38e74c5..d609b4d 100644
--- a/Kvantum/Matcha-Light-Aliz/Matcha-Light-Aliz.kvconfig
+++ b/Kvantum/Matcha-Light-Aliz/Matcha-Light-Aliz.kvconfig
@@ -61,7 +61,7 @@ dialog_button_layout=0
 intensity=1.00
 no_inactiveness=false
 no_window_pattern=false
-opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx
+opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx,calamares
 reduce_menu_opacity=0
 reduce_window_opacity=0
 respect_DE=true
diff --git a/Kvantum/Matcha-Light-Azul/Matcha-Light-Azul.kvconfig b/Kvantum/Matcha-Light-Azul/Matcha-Light-Azul.kvconfig
index 3816e1b..dda24ea 100644
--- a/Kvantum/Matcha-Light-Azul/Matcha-Light-Azul.kvconfig
+++ b/Kvantum/Matcha-Light-Azul/Matcha-Light-Azul.kvconfig
@@ -61,7 +61,7 @@ dialog_button_layout=0
 intensity=1.00
 no_inactiveness=false
 no_window_pattern=false
-opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx
+opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx,calamares
 reduce_menu_opacity=0
 reduce_window_opacity=0
 respect_DE=true
diff --git a/Kvantum/Matcha-Light/Matcha-Light.kvconfig b/Kvantum/Matcha-Light/Matcha-Light.kvconfig
index 5c9ae39..ae4d949 100644
--- a/Kvantum/Matcha-Light/Matcha-Light.kvconfig
+++ b/Kvantum/Matcha-Light/Matcha-Light.kvconfig
@@ -61,7 +61,7 @@ dialog_button_layout=0
 intensity=1.00
 no_inactiveness=false
 no_window_pattern=false
-opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx
+opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx,calamares
 reduce_menu_opacity=0
 reduce_window_opacity=0
 respect_DE=true
diff --git a/Kvantum/Matcha/Matcha.kvconfig b/Kvantum/Matcha/Matcha.kvconfig
index c4c0cd6..96e2cc4 100644
--- a/Kvantum/Matcha/Matcha.kvconfig
+++ b/Kvantum/Matcha/Matcha.kvconfig
@@ -61,7 +61,7 @@ dialog_button_layout=0
 intensity=1.00
 no_inactiveness=false
 no_window_pattern=false
-opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx
+opaque=kaffeine,kmplayer,subtitlecomposer,kdenlive,vlc,smplayer,smplayer2,avidemux,avidemux2_qt4,avidemux3_qt4,avidemux3_qt5,kamoso,QtCreator,VirtualBox,trojita,dragon,digikam,lyx,calamares
 reduce_menu_opacity=0
 reduce_window_opacity=0
 respect_DE=true
